src/main/java/br/com/mercadolivre/loja/user/UserController.java

```diff
@@ -1,31 +1,38 @@
 package br.com.mercadolivre.loja.user;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
 import org.springframework.validation.BindingResult;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
 
 @Controller()
```

Comment:
Em Java, não é comum colocarmos o `()` numa anotação sem mudança de parâmetros. Acho que é assim no Nest, né?

```suggestion
@Controller
```

Reply:
Não lembro com exatidão, acho que lá é necessário usar. Aqui fui no automático.